app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -485,6 +491,8 @@ private void hideInfoLayoutArcPointer() {
 
     @Override
     public void onDestroy() {
+        LocalBroadcastManager.getInstance(this).sendBroadcast(new Intent(ACTION_STOP));
+
```

Comment:
This intent is used to notify to MainActivity that the service is stopping so it need to update the UI (launch button). This happens when the user stops the service from the notification action while the GoIV app is running below the notification bar!